UnityProject/Assets/Scripts/Objects/Pipes/Devices/Scrubber.cs

```diff
@@ -393,7 +393,14 @@ private void UpdatePowerUsage()
 		};
 
 		private AcuSample atmosphericSample = new AcuSample();
-		AcuSample IAcuControllable.AtmosphericSample => atmosphericSample.FromGasMix(metaNode.GasMixLocal);
+		AcuSample IAcuControllable.AtmosphericSample
```

Comment:
```suggestion
		AcuSample? IAcuControllable.AtmosphericSample
```

I remember in a recent PR  that we support nullables, right? We should totally use them whenever possible.

Reply:
We don't entirely support them. If you add a `?` here it will just trigger Jetbrains to replace it with the `[CanBeNull]` annotation.

Reply:
that's what I thought initially, but I saw a PR recently that had ``?``. I think it was @Bod9001